class.jetpack-debugger.php

```diff
@@ -294,6 +294,29 @@ public static function jetpack_debug_display_handler() {
 							$subject_line
 						);
 					?>
+					<div id="category_div" class="formbox">
+						<label class="h" for="category"><?php esc_html_e( 'What do you need help with?', 'jetpack' ); ?></label>
+						<select name="category" id="category">
+						<?php
+						/**
+						 * Set up an array of ticket categories.
+						 * (reasons why a user would contact us.)
+						 */
+						$categories = array(
+							'Connection' => esc_html__( "I'm having trouble connecting Jetpack to WordPress.com", 'jetpack' ),
+							'Billing'    => esc_html__( 'I have a billing or plans question', 'jetpack' ),
+							'Backups'    => esc_html__( 'I need help with backing up or restoring my site', 'jetpack' ),
+							'Security'   => esc_html__( 'I have security concerns / my site is hacked', 'jetpack' ),
+							'Priority'   => esc_html__( "My site is down / I can't access my site", 'jetpack' ),
+							'Other'      => esc_html__( 'Something Else' ),
```

Comment:
Good call. Fixed in ab831c5b0e33f822026bef6871f5109d22bd2f79

I'll keep editing this PR tomorrow, I want to switch from a dropdown to radio buttons, like on the contact form on WordPress.com.